app.py

```diff
@@ -1,14 +1,22 @@
-from selenium import webdriver
+from selenium import webdriver, common
 from webdriver_manager.chrome import ChromeDriverManager
 from random import randint
 import time
-import winsound
+import pyttsx3
```

Comment:
This will need to be added to requirements.txt as well.

Reply:
whoops forgot to pip freeze